### PR TITLE
[sqlcipher] Update to 4.4.1

### PR DESCRIPTION
**Pull request by SvenPStarFinanz**
Updated sqlcipher to 4.4.1, which itself followed sqlite to 3.33.0.

### CODE REVIEW

**Review by NancyLi1013 — COMMENTED** [review]
Could you please also bump the version in `vcpkg.json`file?

**Comment by SvenPStarFinanz** [comment]
Oops! Done.

File: ports/sqlcipher/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sqlcipher",
-  "version-string": "4.4.0",
+  "version-string": "4.4.1",
   "port-version": 1,
```
Comment (NancyLi1013) [inline ports/sqlcipher/vcpkg.json:4]:
```suggestion
```
Since this is a new version, port-version is no longer needed here.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
Have you tested these features? Can they work fine?

**Comment by SvenPStarFinanz** [comment]
I have tested (with sqlitebrowser.org), for me they work.

**Comment by NancyLi1013** [comment]
LGTM now and thanks for your PR @SvenPStarFinanz.

**Comment by strega-nil** [comment]
Thanks @SvenPStarFinanz !

**Pull request merged**